usr/share/lib/ipa/tests/SLES/test_sles_install_migration.py

```diff
@@ -1,2 +1,3 @@
 def test_sles_install_migration(InstallZypperPackage):
-    assert InstallZypperPackage('zypper-migration-plugin')
+    result = InstallZypperPackage('zypper-migration-plugin')
+    assert result == 0
```

Comment:
Double check the return value if zypper if the package is already installed. The migration plugin is already installed in some images I think, if not we will eventually get to the point where it is installed everywhere.

Reply:
It still returns result code of 0 if package already installed along with 'Nothing to do.'.